src/lib/stores.ts

```diff
@@ -11,3 +11,6 @@ export const debug = (() => {
     (v) => (_debug = dev && v)
   );
 })();
+
+export const deckNames = writable<string[]>([]);
+export const currentDeck = writable<string>("rpg");
```

Comment:
See comments on #25

Reply:
I interpreted this comment as hobble the deckchanger button and put in WIP placeholder values until phase 2?

Reply:
I left these here for not, but disabled the Deckchanger button in the toolbar. is that in line with what you were thinking?

Reply:
I think it's good to merge! I can explore more post-deadline.